src/pages/brain/page-details.scss

```diff
@@ -0,0 +1,33 @@
+@import "react-notion-x/src/styles.css";
```

Comment:
we put global styles like this into styles.ts. That's css in js with support for the design system